src/main/java/bio/terra/common/stairway/StairwayComponent.java

```diff
@@ -137,4 +137,8 @@ public Stairway get() {
   public StairwayComponent.Status getStatus() {
     return status.get();
   }
+
+  private String getClusterName() {
+    return String.format("%s-%s", kubeService.getNamespace(), stairwayProperties.getClusterNameSuffix());
+  }
```

Comment:
I don't think I have all the context on this change, but does this logic to construct a K8S cluster name by concatenating the namespace and a suffix apply to all services that use Stairway (e.g. will it apply to TDR also or only AP-style deployments)?